test/clj/doit/todo/api_test.clj

```diff
@@ -1,54 +1,80 @@
 (ns doit.todo.api-test
```

Comment:
You should also add tests to check program behavior on sign-in success and failure.